recipe/meta.yaml

```diff
@@ -11,41 +11,53 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  noarch: python
-  number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
-
-requirements:
-  host:
-    - python >=3.6
-    - pip
-
-  run:
-    - python >=3.6
-    - cycler
-    - databroker
-    - doct
-    - event-model >=1.10.0
-    - historydict
-    - ipython
-    - lmfit
-    - matplotlib
-    - msgpack-python
-    - msgpack-numpy
-    - numpy
-    - ophyd
-    - pyzmq
-    - super_state_machine
-    - toolz
-    - tqdm
-    - zict
+  number: 1
 
 test:
   imports:
     - bluesky
     - bluesky.plans
     - bluesky.callbacks
 
+outputs:
+  - name: bluesky-base
+    build:
+      noarch: python
+      script: python -m pip install . --no-deps -vv
+    requirements:
+      host:
+        - python >=3.6
+        - pip
+      run:
+        - python >=3.6
+        - cycler
+        - event-model >=1.10.0
+        - historydict
+        - msgpack-numpy
+        - msgpack-python
+        - numpy
+        - super_state_machine
+        - toolz
+        - tqdm >=4.44
```

Comment:
The original version did not have this pin. What's the motivation for that?

Reply:
It's pinned in bluesky requirements.txt, thought I would add it here. Can remove if desired.

Reply:
https://github.com/bluesky/bluesky/blob/master/requirements.txt#L9

Reply:
It's fine, let's keep it with the pin.